ggshield/core/config/user_config.py

```diff
@@ -30,6 +30,7 @@
 _IGNORE_KNOWN_SECRETS_KEY = "ignore_known_secrets"
 
 GHSA_ID_PATTERN = re.compile("GHSA(-[a-zA-Z0-9]{4}){3}")
+CVE_ID_PATTERN = re.compile("CVE-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4,}")
```

Comment:
according to their [doc](https://cve.mitre.org/cve/identifiers/syntaxchange.html), the CVE pattern will only have digits so something like `re.compile(r"CVE-\d{4}-\d{4,}")` should do, doesn't it?